infrastructure/network/netadapter/server/grpcserver/protowire/rpc.proto

```diff
@@ -532,3 +532,21 @@ message NotifyVirtualSelectedParentBlueScoreChangedResponseMessage {
 message VirtualSelectedParentBlueScoreChangedNotificationMessage {
   uint64 virtualSelectedParentBlueScore = 1;
 }
+
+// BanRequestMessage bans the given ip.
```

Comment:
Please also run the documentation generator.
See: infrastructure/network/netadapter/server/grpcserver/protowire/README.md